src/components/comment/commentContext.js

```diff
@@ -0,0 +1,133 @@
+import React, { createContext, useContext, useState } from 'react';
+import _ from 'lodash';
+
+import * as fetchUtil from '../../util/fetch';
+
+/** @type {React.Context<{commentList:Comment[],commentError:Error,getData:()=>void}} */
+const commentContext = createContext();
+
+/** This is a react component which you wrap your entire application
+ * to provide a "context", meaning: data you can access anywhere in the app.
+ *
+ * @param {object} props
+ * @param {React.ReactChild} props.children
```

Comment:
I'm actually not sure if React.ReactChild is right :/ don't need to fix now, but I gotta look that up.